human-eval-benchmark/script.py

```diff
@@ -0,0 +1,100 @@
+import torch
+if torch.cuda.is_available():
```

Comment:
This `if-else` looks redundant since we're already printing the device name below.

Reply:
removed